web/checks_api.go

```diff
@@ -100,17 +122,38 @@ func ApiCreateChecksCatalogHandler(s services.ChecksService) gin.HandlerFunc {
 // @Success 200 {object} JSONChecksSettings
 // @Failure 404 {object} map[string]string
 // @Router /api/checks/{id}/settings [get]
-func ApiCheckGetSettingsByIdHandler(s services.ChecksService) gin.HandlerFunc {
+func ApiCheckGetSettingsByIdHandler(consul consul.Client, s services.ChecksService) gin.HandlerFunc {
 	return func(c *gin.Context) {
-		id := c.Param("id")
+		clusterId := c.Param("id")
```

Comment:
I wouldn't rename this to `clusterId`. The checks should be agnostic on this.
I understand that right now it is only attached to clusters, but at some point it won't be.

Maybe we should have other extra api call to get the hosts and their defaults settings

Reply:
I didn't know about this future scope and I wish I'd known :joy: anyway, you're right but as of now this endpoint has a lot of bloatware with the whole consul cluster spattered inside, so the neutrality of the thing is way too compromised by the entire code structure I adopted, with this being a tiny detail. Anyway, I can rename this to `resourceId` and expand the TODO item, what do you think?